packages/neon/neon/lib/src/widgets/image.dart

```diff
@@ -328,13 +286,118 @@ class NeonApiImage extends StatelessWidget {
   }
 }
 
+/// Nextcloud URL image.
+///
+/// Extension for [NeonCachedImage] providing a [NextcloudClient] to the caller
+/// to retrieve the image from an URL.
+///
+/// See:
+///  * [NeonCachedImage] for a customized image
+///  * [NeonImageWrapper] for a wrapping widget for images
```

Comment:
Please allign with the suggestions as made above.

Reply:
I'm not sure what you mean?